test/celeritas/global/AlongStep.test.cc

```diff
@@ -175,6 +177,34 @@ TEST_F(Em3AlongStepTest, msc_nofluct)
     }
 }
 
+TEST_F(Em3AlongStepTest, msc_nofluct_finegrid)
```

Comment:
I realized this test needs new CMake code to activate it:
```diff
diff --git a/test/CMakeLists.txt b/test/CMakeLists.txt
index bd88a999..cde8b3a1 100644
--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -335,6 +335,7 @@ if(CELERITAS_USE_Geant4)
       "-Em3*"
       "Em3AlongStepTest.nofluct_nomsc"
       "Em3AlongStepTest.msc_nofluct"
+      "Em3AlongStepTest.msc_nofluct_finegrid"
       "Em3AlongStepTest.fluct_nomsc"
   )
 else()
```
so I'll add that in my "tracking cut" PR which does along-step stuff.

Reply:
Ah good catch, thanks!